README.dev.md

```diff
@@ -80,7 +80,10 @@ virtualenv -p python3 .venv
 .venv/bin/pip3 install -r requirements.txt
 ```
 ### Setup on Windows 10
-First make sure [Python](https://python.org) 3.7 or greater is installed (in the Python installer window be sure to click on "Add Python to PATH"), then open a PowerShell window and enter the following commands:
+First make sure [Python](https://python.org) 3.7 or greater is installed (in the Python installer window be sure to click on "Add Python to PATH").
+
+#### Python virtual environment
+To setup RP2 in a python virtual environment, enter the following commands:
```

Comment:
Can you leave the reference to Powershell? I.e.:  "open a PowerShell window and enter the following commands"